app/views/lead_providers/partnerships/show.html.erb

```diff
@@ -14,6 +14,14 @@
   <% end %>
 <% end %>
 
+<% if @partnership.challenged? %>
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <%= render LeadProviders::Partnerships::ChallengedBanner.new(partnership: @partnership) %>
+    </div>
+  </div>
+<% end %>
```

Comment:
Just out of curiosity, why a separate grid instead of directly above the H1?

Reply:
I think the only reason is that the rest of the view is build this way. This should be visited separately at some point, if it is worth it :)